vue-components/stories/Lookup.stories.ts

```diff
@@ -43,6 +43,26 @@ const vegetableItems = [
 		description: 'vegetable',
 		value: 'Q150463',
 	},
+	{
+		label: 'cassava root',
```

Comment:
Yums, this is making me hungry... :drooling_face:

Reply:
heheh